packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,6 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
```

Comment:
I am applying application on entire calendar control not on table so it just switches the ATs mode when anything inside calendar control is focused. I tested with screen readers and the experience is smooth with table cells and headers being announced properly.